app/ios/Podfile.lock

```diff
@@ -1,5 +1,7 @@
 PODS:
   - Flutter (1.0.0)
+  - flutter_email_sender (0.0.1):
```

Comment:
Ah yes, I suppose we missed testing this on iOS sim on local and checking that in. I suppose the GitHub Actions machine did this, but of course, that's lost to us.

Reply:
I think it is fine to just have the next person commit these changes vs. Sana asking someone to see if cocoa pods changed from her non-mac changes.